### PR TITLE
SNIC -> NAISS

### PR DESCRIPTION
**Pull request by ssjunnebo**

### CODE REVIEW

File: ACKNOWLEDGEMENTS_S.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Services provided by NGI must be acknowledged in all scientific dissemination activities, such as publications, presentations, posters, etc., using the following statement:
-    
-"The authors acknowledge support from the National Genomics Infrastructure in Stockholm funded by Science for Life Laboratory, the Knut and Alice Wallenberg Foundation and the Swedish Research Council, and SNIC/Uppsala Multidisciplinary Center for Advanced Computational Science for assistance with massively parallel sequencing and access to the UPPMAX computational infrastructure."
+
+"The authors acknowledge support from the National Genomics Infrastructure in Stockholm funded by Science for Life Laboratory, the Knut and Alice Wallenberg Foundation and the Swedish Research Council, and NAISS/Uppsala Multidisciplinary Center for Advanced Computational Science for assistance with massively parallel sequencing and access to the UPPMAX computational infrastructure."
```
Comment (aanil) [inline ACKNOWLEDGEMENTS_S.txt:3]: This [page](https://www.naiss.se/policies/acknowledge/) says `Acknowledge both NAISS and SNIC if resources were used before and after 2023-01-01.`

Comment (aanil) [reply ACKNOWLEDGEMENTS_S.txt:3]: There's also a recommended way to acknowlege them [here](https://www.naiss.se/policies/acknowledge/acknowledge-naiss-snic/)

Comment (b97pla) [reply ACKNOWLEDGEMENTS_S.txt:3]: could you please change the Uppsala version as well?

Comment (ssjunnebo) [reply ACKNOWLEDGEMENTS_S.txt]: Done! Pär will bring up the changes at a management meeting tomorrow so let's wait with merging until then.

Comment (ssjunnebo) [reply ACKNOWLEDGEMENTS_S.txt]: Unfortunately we won't have a decision on this until after the deployment on Monday. Let's hold off on deploying this.

**Review by aanil — APPROVED** [review]
👍

**Pull request merged**